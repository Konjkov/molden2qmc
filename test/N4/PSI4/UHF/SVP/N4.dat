#!n4 high ang. mo. test
memory 1000 mb
molecule n4 {
symmetry = c1
1 2
N       0.5200      0.5300      0.5100
N      -0.5200     -0.5200      0.5200
N      -0.5200      0.5200     -0.5200
N       0.5200     -0.5200     -0.5200
}

set {
  basis def2-svp
  reference uhf
  scf_type pk
  molden_write true
}

energy('scf')
