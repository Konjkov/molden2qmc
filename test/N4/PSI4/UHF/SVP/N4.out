    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 0.4.156 Driver

                          Git: Rev {master} 7783818 dirty

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Mon May 16 07:00:31 2016

    Process ID:  12557
    PSI4DATADIR: /usr/local/psi4/share/psi4
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
#!n4 high ang. mo. test
memory 1000 mb
molecule n4 {
symmetry = c1
1 2
N       0.5200      0.5300      0.5100
N      -0.5200     -0.5200      0.5200
N      -0.5200      0.5200     -0.5200
N       0.5200     -0.5200     -0.5200
}

set {
  basis def2-svp
  reference uhf
  scf_type pk
  molden_write true
}

energy('scf')
--------------------------------------------------------------------------

  Memory set to 1000.000 MiB by Python script.

*** tstart() called on vladimir-Aspire-E5-571G
*** at Mon May 16 07:00:34 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              UHF Reference
                        1 Threads,   1000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 1, multiplicity = 2:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           N          0.520000000000     0.527500000000     0.512500000000    14.003074004780
           N         -0.520000000000    -0.522500000000     0.522500000000    14.003074004780
           N         -0.520000000000     0.517500000000    -0.517500000000    14.003074004780
           N          0.520000000000    -0.522500000000    -0.517500000000    14.003074004780

  Running in c1 symmetry.

  Rotational constants: A =      0.55979  B =      0.55648  C =      0.55324 [cm^-1]
  Rotational constants: A =  16782.11936  B =  16682.72627  C =  16585.64680 [MHz]
  Nuclear repulsion =  105.777940020337695

  Charge       = 1
  Multiplicity = 2
  Electrons    = 27
  Nalpha       = 14
  Nbeta        = 13

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: DEF2-SVP
    Number of shells: 24
    Number of basis function: 56
    Number of Cartesian functions: 60
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         56      56       0       0       0       0
   -------------------------------------------------------
    Total      56      56      14      13      13       1
   -------------------------------------------------------

  ==> Integral Setup <==

	Batch   1 pq = [       0,    1596] index = [             0,1274406]
  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory (MB):               715
    Schwarz Cutoff:          1E-12

  Minimum eigenvalue in the overlap matrix is 7.3429304367E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @UHF iter   1:  -200.69256559609218   -2.00693e+02   8.51175e-02 
   @UHF iter   2:  -209.13055773438182   -8.43799e+00   7.03314e-02 DIIS
   @UHF iter   3:  -216.54939113059774   -7.41883e+00   1.22491e-02 DIIS
   @UHF iter   4:  -216.62781528943779   -7.84242e-02   1.08566e-02 DIIS
   @UHF iter   5:  -216.78568631103425   -1.57871e-01   1.81513e-03 DIIS
   @UHF iter   6:  -216.79720586086196   -1.15195e-02   6.44621e-04 DIIS
   @UHF iter   7:  -216.79979893864223   -2.59308e-03   4.27294e-04 DIIS
   @UHF iter   8:  -216.80083947958187   -1.04054e-03   2.61665e-04 DIIS
   @UHF iter   9:  -216.80163185835713   -7.92379e-04   1.94804e-04 DIIS
   @UHF iter  10:  -216.80236051673151   -7.28658e-04   1.47381e-04 DIIS
   @UHF iter  11:  -216.80283365052779   -4.73134e-04   1.15065e-04 DIIS
   @UHF iter  12:  -216.80301012728094   -1.76477e-04   9.99880e-05 DIIS
   @UHF iter  13:  -216.80321706591351   -2.06939e-04   7.98756e-05 DIIS
   @UHF iter  14:  -216.80344512118435   -2.28055e-04   5.41169e-05 DIIS
   @UHF iter  15:  -216.80352826416751   -8.31430e-05   2.96472e-05 DIIS
   @UHF iter  16:  -216.80355359387048   -2.53297e-05   1.77290e-05 DIIS
   @UHF iter  17:  -216.80355754448297   -3.95061e-06   1.60802e-05 DIIS
   @UHF iter  18:  -216.80356054636806   -3.00189e-06   1.60092e-05 DIIS
   @UHF iter  19:  -216.80355968028445    8.66084e-07   1.58932e-05 DIIS
   @UHF iter  20:  -216.80355587407988    3.80620e-06   1.55817e-05 DIIS
   @UHF iter  21:  -216.80354506131610    1.08128e-05   1.31576e-05 DIIS
   @UHF iter  22:  -216.80353539270001    9.66862e-06   8.74614e-06 DIIS
   @UHF iter  23:  -216.80353275583585    2.63686e-06   5.70996e-06 DIIS
   @UHF iter  24:  -216.80353215202788    6.03808e-07   3.56758e-06 DIIS
   @UHF iter  25:  -216.80353217498205   -2.29542e-08   1.81584e-06 DIIS
   @UHF iter  26:  -216.80353220903262   -3.40506e-08   1.46821e-06 DIIS
   @UHF iter  27:  -216.80353225687614   -4.78435e-08   9.26548e-07 DIIS

  ==> Post-Iterations <==

   @Spin Contamination Metric:   1.232635007E-01
   @S^2 Expected:                7.500000000E-01
   @S^2 Observed:                8.732635007E-01
   @S   Expected:                5.000000000E-01
   @S   Observed:                5.000000000E-01

    Orbital Energies (a.u.)
    -----------------------

    Alpha Occupied:                                                       

       1A    -16.158623     2A    -16.096944     3A    -16.096340  
       4A    -16.093245     5A     -2.010613     6A     -1.444405  
       7A     -1.373748     8A     -1.371624     9A     -1.064870  
      10A     -0.957627    11A     -0.913126    12A     -0.911997  
      13A     -0.885055    14A     -0.881029  

    Alpha Virtual:                                                        

      15A     -0.159125    16A     -0.155582    17A     -0.120214  
      18A     -0.043247    19A     -0.024501    20A     -0.021218  
      21A      0.408240    22A      0.537333    23A      0.562714  
      24A      0.563460    25A      0.625912    26A      0.626590  
      27A      0.630484    28A      0.642659    29A      0.644427  
      30A      0.775082    31A      0.776115    32A      0.793850  
      33A      0.796351    34A      0.926335    35A      0.946718  
      36A      0.950479    37A      1.539748    38A      1.543278  
      39A      1.559936    40A      1.778689    41A      1.782887  
      42A      2.040594    43A      2.204292    44A      2.207002  
      45A      2.217644    46A      2.295944    47A      2.303086  
      48A      2.345479    49A      2.560613    50A      2.570268  
      51A      2.971399    52A      3.023538    53A      3.025606  
      54A      3.192615    55A      3.199670    56A      3.239284  

    Beta Occupied:                                                        

       1A    -16.100034     2A    -16.098132     3A    -16.097285  
       4A    -16.096055     5A     -1.996979     6A     -1.363324  
       7A     -1.357841     8A     -1.310037     9A     -0.968191  
      10A     -0.894860    11A     -0.894038    12A     -0.860322  
      13A     -0.854275  

    Beta Virtual:                                                         

      14A     -0.432946    15A     -0.125227    16A     -0.096393  
      17A     -0.093712    18A     -0.022991    19A     -0.019102  
      20A      0.001628    21A      0.470126    22A      0.583488  
      23A      0.583812    24A      0.604450    25A      0.634614  
      26A      0.635346    27A      0.644174    28A      0.645302  
      29A      0.655089    30A      0.786653    31A      0.787230  
      32A      0.794353    33A      0.813450    34A      0.942890  
      35A      0.947455    36A      0.950828    37A      1.566463  
      38A      1.568405    39A      1.578722    40A      1.803274  
      41A      1.806896    42A      2.060677    43A      2.224226  
      44A      2.232477    45A      2.235414    46A      2.327668  
      47A      2.333795    48A      2.338815    49A      2.576825  
      50A      2.586548    51A      3.019999    52A      3.025955  
      53A      3.028749    54A      3.218415    55A      3.225217  
      56A      3.242075  

    Final Occupation by Irrep:
              A 
    DOCC [    13 ]
    SOCC [     1 ]

  Energy converged.

  @UHF Final Energy:  -216.80353225687614

   => Energetics <=

    Nuclear Repulsion Energy =            105.7779400203376952
    One-Electron Energy =                -494.0122815054784269
    Two-Electron Energy =                 171.4308092282645930
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -216.8035322568761103

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.0000      Y:    -0.0000      Z:    -0.0000

  Electronic Dipole Moment: (a.u.)
     X:    -0.3161      Y:    -0.3195      Z:     0.3215

  Dipole Moment: (a.u.)
     X:    -0.3161      Y:    -0.3195      Z:     0.3215     Total:     0.5525

  Dipole Moment: (Debye)
     X:    -0.8033      Y:    -0.8120      Z:     0.8172     Total:     1.4044


  Saving occupied orbitals to File 180.

  UHF NO Occupations:
  HONO-2 :   12  A 1.9740971
  HONO-1 :   13  A 1.9717983
  HONO-0 :   14  A 1.0000000
  LUNO+0 :   15  A 0.0282017
  LUNO+1 :   16  A 0.0259029
  LUNO+2 :   17  A 0.0053144
  LUNO+3 :   18  A 0.0010434


*** tstop() called on vladimir-Aspire-E5-571G at Mon May 16 07:00:36 2016
Module time:
	user time   =       1.65 seconds =       0.03 minutes
	system time =       3.91 seconds =       0.07 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       1.65 seconds =       0.03 minutes
	system time =       3.91 seconds =       0.07 minutes
	total time  =          2 seconds =       0.03 minutes

*** PSI4 exiting successfully. Buy a developer a beer!
